### ISSUE TITLE
Update smallrye-gcp-pubsub

### ISSUE THREAD

**Issue by silviu-negoita** [smallrye/smallrye-reactive-messaging#1471]
Hello,

We are using the very 'hidden' nice connector `smallrye-gcp-pubsub`. I'm saying hidden, because I didn't find documentation ([here](https://smallrye.io/smallrye-reactive-messaging/smallrye-reactive-messaging/2/connectors/connectors.html)) about it, despite the fact that is released and updated in maven central [repo](https://mvnrepository.com/artifact/io.smallrye.reactive/smallrye-reactive-messaging-gcp-pubsub/3.1.0).

 It is working very well, but our infra is a bit more complex, requires a `projectId` configured for each subscription and topic (Long discussion about this, ask me if you need more details about it). Currently the connector is using globally configured project id ( `gcp-pubsub-project-id`)  

Basically I need 1 more configuration per channel(e.g. `projectId`) and some code changes to actually use them when creating publisher and/or subscriber. 

Can I contribute to this? If yes, who can help me to ask some questions?

**Comment by cescoffier** [comment]
This connector has been moved to the "attic." Unfortunately, we didn't have a maintainer, and the rest of the team did not have the knowledge and experience with GCP Pub/Sub. It is unfortunate, but, to keep the project healthy, we had to make that move. However, there is no problem bringing the connector back, especially if someone (like you) volunteers to help us maintain it. 

About your issue, yes, I would prefer to avoid having to use admin clients. It can be an option, but by default, it should be disabled. We did the same kind of things with Kafka, where, initially, we used an AdminClient, but it caused too many issues.
